pages/osx/sed.md

```diff
@@ -5,32 +5,32 @@
 
 - Replace the first occurrence of a string in a file, and print the result:
 
-`sed 's/{{find}}/{{replace}}/' {{filename}}`
+`sed "s/{{find}}/{{replace}}/" {{filename}}`
```

Comment:
I think the `'` is to avoid issues with escape sequences or env vars.

Reply:
Maybe it makes sense to use single quotes for sed/awk and other inlined code. :)